### DISCUSSION TITLE
Server error when checking out

### DISCUSSION THREAD

**Discussion by AmericanTrails** [craftcms/.github#12] [Craft Console] [answered]
I'm trying to upgrade Freeform Lite to Freeform Pro. I'm following the [instructions](https://docs.solspace.com/craft/freeform/v4/setup/installing-updating.html#changing-editions) Solspace but am receiving a "Server Error" message when clicking the "Pay" button. See [screenshot](https://ibb.co/Z14C2yG)

**Comment by angrybrad** [comment]
@AmericanTrails what Craft ID account are you using when you try that purchase? Feel free to send it to support@craftcms.com and reference this discussion if you don't want to share here.

**Reply by AmericanTrails** [reply]
Email sent.

**Comment by angrybrad** [comment, marked as answer]
Due to an incomplete billing address on the card. Filling that out will resolve it as a workaround, but we're working on adding better validation/error handling around this on our end, too.

**Reply by timkelty** [reply]
Validation is now displayed in this scenario.
